.ci/ansible/playbook.yml

```diff
@@ -35,7 +35,7 @@
     - role: andrewrothstein.kind
     - role: gantsign.golang
       vars:
-        golang_version: 1.16.3
+        golang_version: 1.17
```

Comment:
We need a way to pass `.go-version` file to here

Reply:
@adam-stokes what do you think about having a Make goal reading `.go-version` to install Go using Andrew Kroh's https://github.com/andrewkroh/gvm? Therefore, we won't duplicate Go versions in multiple places:

```shell
eval "$(gvm $(cat .go-version))"
```